decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -28,17 +28,18 @@ def available_orders
         end
 
         def default_order
-          if current_settings.votes_blocked?
-            detect_order("most_voted")
-          else
-            "random"
-          end
+          return detect_order("most_voted") if votes_blocked?
+          "random"
```

Comment:
I generally dislike early returns when the conditional branches have equal importance. I prefer to limit them to error checking, edge cases and things like that. Also, with the early return you get more code coverage false positives. For example, even if we wouldn't test the `most_voted` order, the whole method will appear as covered as long as we test the `random` order.

Anyways, just a minor style nitpick.